src/mono/mono/component/mini-wasi-debugger.c

```diff
@@ -0,0 +1,172 @@
+#include <glib.h>
+#include <mono/mini/debugger-agent-external.h>
+#include <mono/component/debugger-engine.h>
+#include <mono/metadata/components.h>
+
+#ifdef HOST_WASI
+static int conn_fd;
+static int log_level = 1;
+static int retry_receive_message = 100;
+static int connection_wait = 1000;
```

Comment:
I think it's actually a value in **micro**seconds, because later on we compare `time_in_milliseconds` against `connection_wait / 1000`.

But also it's a bit misleading because it rounds `connection_wait / 1000` down to a whole number of milliseconds, so any value < 1000 is equivalent to zero.

Would it be clearer to change this to `connection_wait_ms` and then not do the `/ 1000` later?